### DISCUSSION TITLE
WARNINGS AND ERRORS

### DISCUSSION THREAD

**Discussion by RRCHA000** [Chia-Network/chia-blockchain#6083] [Support]
Hello!

Can you help me with these informations. Is it necessary to do anything to eliminate that?

 WARNING  Wasn't able to add transaction with id f8c5a75bf26fa98f30e057137932f3d1aff826f8dea415307bf7d40ca5d63bd9, status 3 error: Err.DOUBLE_SPEND
2021-05-24T10:41:49.901 full_node full_node_server        : ERROR    Exception:  <class 'concurrent.futures._base.CancelledError'>, closing connection {'host': '195.239.229.166', 'port': 8444}. Traceback (most recent call last):
  File "chia\server\server.py", line 531, in api_call
  File "asyncio\tasks.py", line 435, in wait_for
concurrent.futures._base.CancelledError

**Comment by tiny0little** [comment]
1st line - DOUBLE_SPEND is a attack of spending same coin twice, detected and prevented
2nd line - a traceback, usually does not happen, possibly related to the node who were trying to do DOUBLE_SPEND, connection was closed

this is normal, you don't need to do anything about it

**Comment by BrandtH22** [comment]
@RRCHA000 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
